go.mod

```diff
@@ -104,6 +104,7 @@ require (
 	gopkg.in/gcfg.v1 v1.2.0
 	gopkg.in/square/go-jose.v2 v2.2.2
 	gopkg.in/yaml.v2 v2.4.0
+	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
```

Comment:
any way we can unify on one version?

Reply:
@thockin the utilities we are writing to cleanup OWNERS files depend on yaml.v3 ( as the yaml.v2 does not have the features needed - say `yaml.Node` and other things doc'ed here https://ubuntu.com/blog/api-v3-of-the-yaml-package-for-go-is-available ). So trying to make sure we use the same version (yaml.v3) for the formatter and the thing we use for cleaning up OWNERS ( some WIP in https://github.com/dims/maintainers/blob/main/yaml_utils.go ). Also there are other PRs where we are trying to move things to yaml.v3 from v2 ... so hoping we get there!

Reply:
we already had deps on yaml.v3, this just uses it from the root module so it shows up in this go.mod file (note that there are no new vendor files in this PR)

there's a compatibility issue between yaml.v3 and yaml.v2 that needs to be resolved before we can move completely to v3 (https://github.com/go-yaml/yaml/issues/720, proposed fix in https://github.com/go-yaml/yaml/pull/753)